plugins/kubernetes-common/api-report.md

```diff
@@ -194,10 +194,14 @@ export interface KubernetesFetchError {
 export interface KubernetesRequestBody {
   // (undocumented)
   auth?: {
-    google?: string;
+    google: string;
   };
   // (undocumented)
   entity: Entity;
+  // (undocumented)
+  oidc?: {
+    [key: string]: string;
+  };
```

Comment:
Can this go inside the `auth` field?

Reply:
The reason this was handled on a separate property is that there is already some overlap in the names of the authentication methods for kubernetes authProviders. If this fields were included in the "auth" field, there would be no distinction between `authProvider: google` and `authProvider: oidc, oidcTokenProvider: google`, and they behave differently, since the former uses access_tokens to authenticate to the cluster and the later uses id_tokens

Reply:
Oh sorry, I think I misunderstood. Do you mean to add the "oidc" option under auth? I can definitely do that

Reply:
So it is currently:

```typescript
export interface KubernetesRequestBody {
  auth?: {
    google: string;
  };
  oidc?: {
    [key: string]: string;
  };
  entity: Entity;
}
```

and I think:

```typescript
export interface KubernetesRequestBody {
  auth?: {
    google?: string;
     oidc?: {
        [key: string]: string;
      };
  };
  entity: Entity;
}
```

Might make more sense, how possible do you think that would be?

Reply:
I just pushed a new commit with this changes, thanks for the suggestion!!